### DISCUSSION TITLE
Support at least some form on non-proprietary community communication

### DISCUSSION THREAD

**Discussion by toastal** [nesbox/TIC-80#2489] [Ideas]
TIC-80 seems interesting to me, but all options to talk with people are proprietary: Microsoft GitHub (code forge & forum), Telegram, Discord, Twitter. For an open source project to have only proprietary tooling doesn’t feel good. It would be nice to feel welcomed with an IRC room, XMPP MUC, Matrix Space, Zulip, Discourse, or otherwise instead of needing to hand over my communication data & agree to the ToS of these corporations.

**Comment by joshgoebel** [comment]
This might sound like a reasonable request but _the last thing we need is more fragmentation_. IMHO.

> doesn’t feel good

This is obviously a matter of opinion.

**Reply by toastal** [reply]
If fragmentation is the problem, then I would suggest choosing a open base platform & using gateways/bridges/mirrors to the proprietary ones so when the rug is finally pulled (_enshitification_ seems to be in the vogue word right now) or a user is banned to to US sanctions against their country or whatever, there is a community-controlled option open to anyone at the bottom. As it stands all of these options require agreeing to the ToS & data collection of corporations—which doesn’t sound very open or welcoming.

For chat there are gateways/bridges to IRC, XMPP, Matrix from Telegram, Discord. For code forges, Git is a DVCS where D is for distributed & a lot of non-corporate services can mirror themselves back to Microsoft GitHub. For forums… well, nothing can bridge the gap to this platform, Microsoft GitHub Discussions, but the rest could covered.

> Choosing proprietary tools and services for your free software project ultimately sends a message to downstream developers and users of your project that freedom of all users—developers included—is not a priority.
>
> — Matt Lee, https://www.linuxjournal.com/content/opinion-github-vs-gitlab

**Reply by Weeppiko** [reply]
I don't know about Telegram, but FYI the Discord server was not created and it's not controlled by the project's owner, and it only exists because an user decided to create it, so having a bridge to IRC or anything else it's completely up to people that handles it.
